samples/dashboards/edit-mode/lang/demo.js

```diff
@@ -11,7 +11,7 @@ const norwegianEditMode = {
     chartType: 'Graf type',
     pointFormat: 'Punkt format',
     confirmDestroyRow: 'Vil du ødelegge raden?',
-    confirmDestroyCell: 'Vil du ødelegge cellen?',
+    confirmDestroyrenderTo: 'Vil du ødelegge cellen?',
```

Comment:
We shouldn't change that

Reply:
I fixed it in many places, but I missed this reference. Thanks!